frontend/src/scenes/surveys/SurveyEditQuestionRow.tsx

```diff
@@ -278,6 +278,20 @@ export function SurveyEditQuestionGroup({ index, question }: { index: number; qu
                                                                 Add open-ended choice
                                                             </LemonButton>
                                                         )}
+                                                        <LemonField name="shuffleOptions" className="mt-2">
+                                                            {({
+                                                                value: shuffleOptions,
+                                                                onChange: toggleShuffleOptions,
+                                                            }) => (
+                                                                <LemonCheckbox
+                                                                    checked={!!shuffleOptions}
+                                                                    label="Shuffle Options"
+                                                                    onChange={(checked) =>
+                                                                        toggleShuffleOptions(checked)
```

Comment:
minor nit, it's kea-forms convention to retain value/onChange as the name here, makes it clear when it's a form-controlled component, vs. when we're doing something custom. Makes it easier to read.